### ISSUE TITLE
VS Code cannot use Go-Mono because nameID 2 is set to "Book" instead of "Regular"

### ISSUE THREAD

**Issue by murphym18** [ryanoasis/nerd-fonts#594]
#### 🎯 VS Code cannot use Go-Mono because nameID 2 is set to "Book" instead of "Regular"

VS Code isn't compatible with `Go Mono Nerd Font Complete.ttf` because name ID 2 has the value `Book`.

To work around this issue, I used the [ttx tool](https://fonttools.readthedocs.io/en/latest/ttx.html) to convert the font to xml. Then I replaced both occurrences of `Book` with `Regular`. After that I ran `ttx` again to convert it back to `ttf` and installed the font.

#### 🔧 Your Setup

- I'm using the font `Go Mono Nerd Font Complete.ttf`
- VS Code version `1.54.3`
- Pop_OS! 20.04 (based Ubuntu 20.04)

#### ★ Related
- Issue #332 was very helpful. Though it was only a starting point. 
- [This doc](https://docs.microsoft.com/en-us/typography/opentype/spec/name#name-ids) from Microsoft about name IDs was also helpful.

**Comment by Finii** [comment]
```
$ ttfdump -t name Go\ Mono\ Nerd\ Font\ Complete.ttf
[...]
Name table   2.  PlatformID:     1
                 EncodingID:     0
                 LanguageID:     0
                 NameID:         2
                 Length:         4
                 Offset:         259
                 42 6f 6f 6b                    > Book
```
```
ttfdump -t name ../../../../src/unpatched-fonts/Go-Mono/Regular/Go-Mono.ttf
[...]
Name table   2.  PlatformID:     1
                 EncodingID:     0
                 LanguageID:     0
                 NameID:         2
                 Length:         7
                 Offset:         72
                 52 65 67 75 6c 61 72           > Regular
```

**Comment by Finii** [comment]
I believe this is fixed ...

```
$ ./gotta-patch-em-all-font-patcher\!.sh -jc /Go-Mono
# [Nerd Fonts]  Filter given, limiting search and patch to pathname pattern '/Go-Mono'
# [Nerd Fonts]  Total source fonts found: 4
# [Nerd Fonts]  Release timestamp is Sun, 15 Jan 2023 16:50:15 +0100
# [Nerd Fonts]  Processing font 1/4
# [Nerd Fonts]  Processing font 2/4
# [Nerd Fonts]  Processing font 3/4
# [Nerd Fonts]  Processing font 4/4
Nerd Fonts Patcher v2.3.0-RC (3.4.3) executing
Nerd Fonts Patcher v2.3.0-RC (3.4.3) executing
Nerd Fonts Patcher v2.3.0-RC (3.4.3) executing
Redistributing line gap of 393 (196 top and 197 bottom)
Redistributing line gap of 393 (196 top and 197 bottom)
Redistributing line gap of 393 (196 top and 197 bottom)
Nerd Fonts Patcher v2.3.0-RC (3.4.3) executing
Redistributing line gap of 393 (196 top and 197 bottom)
Done with Patch Sets, generating font...
   Go Mono Nerd Font Complete
   \===> '/home/fini/git/nerd-fonts/check-fonts/Go-Mono/Regular/complete/Go Mono Nerd Font Complete.ttf'
```
```
$ ttfdump -t name '/home/fini/git/nerd-fonts/check-fonts/Go-Mono/Regular/complete/Go Mono Nerd Font Complete.ttf'
[...]
Name table   2.	 PlatformID:	 1
		 EncodingID:	 0
		 LanguageID:	 0
		 NameID:	 2
		 Length:	 7
		 Offset:	 265
		 52 65 67 75 6c 61 72           > Regular
```

Fixed via e97e7959d6f see #1046
